core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -213,18 +220,20 @@ object SparkEnv extends Logging {
       bindAddress: String,
       advertiseAddress: String,
       port: Int,
-      isDriver: Boolean,
```

Comment:
I had to modify this to avoid going over the 10 argument limit the style checker imposes. Which is also a reason why the key is not stored in the SparkEnv itself.